application/models/Permission.php

```diff
@@ -720,15 +724,35 @@ private static function comparePermissionTitle($aApermission, $aBpermission)
     public function getUserId($iUserID = null)
     {
         if (empty($iUserID)) {
-            if (Yii::app() instanceof CConsoleApplication) {
-                /* Alt : return 1st forcedAdmin ? */
+            if (App() instanceof CConsoleApplication) {
```

Comment:
Wondering, what's the difference in between App() and Yii::app(). I guess there is a slight difference...

Reply:
No diiference App is a shortcut for Yii:app